README.md

```diff
@@ -144,12 +138,26 @@ Do you need to know how to use the library in certain scenarios?
 
 We have gathered all the examples in a separate folder and they can be found under the [examples folder](./examples). 
 
+## Versioning and maintenance
+As of version 1, Modelina has a very strict set of changes we are allowed to do before it requires a major version change. In short, any changes that change the generated outcome are not allowed as it's a breaking change for the consumer of the generated models. 
+
+Here is a list of changes we are allowed to do that would not require a breaking change:
+- Adding new features (that do not change existing output), such as generators, presets, input processors, etc.
+- Change existing features, by providing options that default to current behavior.
```

Comment:
I guess this is breaking change or I don't quite understand this sentence, can it be changed?

Reply:
Ok, fine, however I wonder if we should treat `Change existing features, by providing options that default to current behavior.` as breaking change, because it exactly means, that changed function will change output using changed options - and this can also be understood as changing more than just presets or generators etc.

Reply:
But where is the breaking change? 🤔

Because the API of modelina does not have any breaking changes, because it defaults to current behaviour, and the generated models does not contain any breaking functionality 🤔

Reply:
Ok, I understand it differently, but it can be, if you think it's ok.

Reply:
I mean, this text should be easily understood by others, not me 😆 

What about something like.

```suggestion
- Change existing features, by providing options that default to current behavior. This could be a preset that adapts the output based on options, as long as the API of Modelina and the API of the generated models does not have any breaking changes.
```

Reply:
ok, better :)